daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -248,6 +248,9 @@ private[archive] class DecodeV1(minor: LV.Minor) extends Decode.OfPackage[PLF.Pa
         lfDataType.getSerializable,
         ImmArray(params).map(decodeTypeVarWithKind),
         lfDataType.getDataConsCase match {
+          case PLF.DefDataType.DataConsCase.SYNONYM =>
+            //assertSince(LV.Features.type_syn, "DefDataType.DataCons.Synonym") //TODO(NICK)
```

Comment:
Please a reference to the issue. 
something like:
```
  // FIXME https://github.com/digital-asset/daml/issues/3616
  // Comments
```

Reply:
ok

Reply:
done